contracts/oracle/src/lib.rs

```diff
@@ -259,6 +283,13 @@ impl Contract {
         self.used_identities.contains(&normalised_id)
     }
 
+    pub fn get_required_sbt_mint_deposit(is_verified_kyc: bool) -> Balance {
+        if is_verified_kyc {
+            return MINT_TOTAL_COST;
+        };
+        MINT_TOTAL_COST + MINT_COST
+    }
```

Comment:
do we need this function?

BTW, we should use `MINT_TOTAL_COST_WITH_KYC`.

Reply:
This way is easier for the front end app to know what exact deposit to attach

Reply:
DONE

Reply:
I don't think frontend should query SC all the time - it will make it a bit clunky.
Can we just hardcode it?

Reply:
cc: @Megha-Dev-19

Reply:
let's keep it here, but let's try to avoid querying it in the UI